core/common/src/LocalDate.kt

```diff
@@ -13,7 +13,7 @@ import kotlinx.serialization.Serializable
  * The date part of [LocalDateTime].
  *
  * This class represents dates without a reference to a particular time zone.
- * As such, these objects may denote different spans of time in different time zones: for someone in Berlin,
+ * As such, these objects may denote different periods in different time zones: for someone in Berlin,
```

Comment:
This is probably a correct change, but this could interfere with us already having the concept of a "period" in the library: our "periods" are things like "1 day, 2 hours", without an established starting point, but here, the word "period" is used in the sense of "the period between 2020 and 2021", not "the period of 4 years". Is there some other phrasing that's natural but doesn't interfere with the terminology we establish?

Reply:
I see, maybe 'time intervals' or 'time frames' would fit better here